source/languages/en/riak/community/product-advisories/dvv-lastwritewins.md

```diff
@@ -45,12 +45,13 @@ In all bucket types where `last_write_wins` is being used, this issue can be avo
 1. Fetch the patch [ZIP
 file](https://github.com/basho/basho_docs/raw/riak/2.1.1/source/data/dvv_lww_buckets.beam.zip)
 1. Uncompress the patch ZIP file.
-1. Place the beam file in your basho_patches directory (»riak_install_dir>/lib/basho_patches«).
-1. Execute the following commands from the /bin directory of your Riak installation: 
+1. Place the beam file in your basho-patches directory (»riak_install_dir«/lib/basho-patches).
+1. If the beam file is not already owned by the user that Riak is running
```

Comment:
"If the beam file is not already owned by the user that Riak is running under, change the ownership to the user Riak is running under."